### ISSUE TITLE
Error: Template expansion error: time is not a function

### ISSUE THREAD

**Issue by Delivator** [drovp/rename#1]
**OS:** Windows 10 Pro 10.0.19044
**App:** 0.4.0 x64

Hey, I got this error:

```
Error: Template expansion error: time is not a function
    at createRenameTable (C:\Users\Delivator\AppData\Roaming\Drovp\plugins\node_modules\@drovp\rename\dist\processor.js:3892:13)
    at async processor_default (C:\Users\Delivator\AppData\Roaming\Drovp\plugins\node_modules\@drovp\rename\dist\processor.js:3951:23)
    at async process.<anonymous> (C:\Users\Delivator\AppData\Local\Programs\drovp\resources\app\thread.js:4:5473)
```

With this payload:

```json
{
  "inputs": [
    {
      "id": "0nff9mki12",
      "created": 1659561932248,
      "kind": "file",
      "path": "C:\\Users\\Delivator\\Downloads\\Drovp-x64.exe",
      "size": 62630918,
      "type": "exe"
    }
  ],
  "options": {
    "template": "${filename}_${time().format('YYYY-MM-DD_HH-mm')}.${ext}",
    "expandDirectories": false,
    "overwrite": false,
    "preview": false,
    "sorting": "disabled",
    "onMissingMeta": "disabled",
    "replacement": "!",
    "maxLength": 100,
    "emit": false
  },
  "id": "zu1kk25ub4wg"
}
```

**Comment by tomasklaen** [comment]
Yes I know about this one. I'm working on a big encode update, then I have a drovp 0.5.0 ready to go, and then I'll work on this one. *Should* be this week.

**Comment by tomasklaen** [comment]
Fixed in the new 4.0.
